src/pages/LoginView.jsx

```diff
@@ -78,13 +95,13 @@ function LoginView() {
               <Input 
```

Comment:
We shouldn't let users who are already logged in get to this page or the register page. While there isn't an explicit link to those pages once you're logged in, a user can still get to them via the paths in the browser window. You could add a check that works in conjunction with the UserProvider context, but I think it might be best to handle that in a separate issue.

Reply:
I'll make an issue for that now

Reply:
https://github.com/oneleif/olWebsite-React/issues/85